test.sh

```diff
@@ -10,7 +10,7 @@ fi
 # Order doesn't matter. Note: godep-restore is specifically left out of the
 # defaults, because we don't want to run it locally (would be too disruptive to
 # GOPATH).
-RUN=${RUN:-vet fmt migrations unit integration errcheck}
+RUN=${RUN:-vet fmt migrations unit coverage integration errcheck}
```

Comment:
@briansmith Apologies, it was not intentional. I noticed that it was running on local runs yesterday as well and submitted a fix https://github.com/letsencrypt/boulder/pull/2750 - once that is merged it should stop running coverage outside of CI by default.